src/components/Input/index.ts

```diff
@@ -0,0 +1,4 @@
+import Input from './Input';
```

Comment:
можно написать вот так

```suggestion
export { default } from './Input';
```

Reply:
Да, всегда так делал, но airbnb-eslint после какого-то обновления начал кровить на такие реэкспорты, [тут](https://github.com/airbnb/javascript/issues/2500) вот решают проблему